pkg/envtest/server.go

```diff
@@ -255,6 +259,13 @@ func (te *Environment) Start() (*rest.Config, error) {
 	te.CRDInstallOptions.ErrorIfPathMissing = te.ErrorIfCRDPathMissing
 	crds, err := InstallCRDs(te.Config, te.CRDInstallOptions)
 	te.CRDs = crds
+	if err != nil {
```

Comment:
small nit, swap line 261 and 262-264. even if this doesn't break now, it could break in the futrue based on the te struct changing it's guarantees.